assets/js/googlesitekit/components-gm2/Checkbox.js

```diff
@@ -45,15 +45,26 @@ export default function Checkbox( props ) {
 		onKeyDown,
 		loading,
 		alignLeft,
+		onClick,
 	} = props;
 
+	const extraProps = {};
+
+	if ( onClick ) {
+		extraProps.onClick = onClick;
+		extraProps.onKeyDown = () => {};
+		extraProps.tabIndex = 0;
+		extraProps.role = 'button';
+	}
+
 	return (
 		<div className="mdc-form-field">
 			<div
 				className={ classnames( 'mdc-checkbox', {
 					'mdc-checkbox--disabled': disabled,
 					'mdc-checkbox--align-left': alignLeft,
 				} ) }
+				{ ...extraProps }
```

Comment:
Optionally allowing `onClick` on `div.mdc-checkbox`. In our scenario, we're doing this to prevent opening the metric accordion on checkbox click. See:

https://github.com/google/site-kit-wp/blob/cb05c080fccc1655021bfb6f2f89a22e7df761ac/assets/js/components/KeyMetrics/MetricsSelectionPanel/Metrics.js#L100-L109

Reply:
Hey @nfmohit, if making changes to the `Checkbox` (or any component which has a GM3 counterpart when we have more of them), please can you either make a corresponding change to the GM3 version to keep it in sync, or create a followup issue to do so?